### DISCUSSION TITLE
使用某几个插件后看不到入口了qwq，删掉或关油猴都没解决，不知道怎么办

### DISCUSSION THREAD

**Discussion by mldewhite** [the1812/Bilibili-Evolved#4019] [综合讨论 / General]
![哔哩哔哩 (゜-゜)つロ 干杯~-bilibili 和另外 4 个页面 - 个人 - Microsoft​ Edge Dev 2023_2_21 19_27_08](https://user-images.githubusercontent.com/125762030/220333021-c828b1f8-25e6-421b-aefe-c8d547fe6ff3.png)

**Comment by Geeyun-JY3** [comment]
我的建议是先翻翻墙试试能不能加载出来。

顺便提醒一句，反馈最好带上脚本版本、浏览器版本、装的是 jsDelivr 源还是 GitHub 源等信息。
